Lib/test/test_asyncio/test_subprocess.py

```diff
@@ -182,6 +183,29 @@ def test_kill(self):
         else:
             self.assertEqual(-signal.SIGKILL, returncode)
 
+    def test_kill_issue43884(self):
+        import subprocess
+        blocking_shell_command = f'{sys.executable} -c "import time; time.sleep(100000000)"'
+        creationflags = 0
+        if sys.platform == 'win32':
+            # On windows create a new process group so that killing process
+            # kills the process and all its children.
+            creationflags = subprocess.CREATE_NEW_PROCESS_GROUP
+        proc = self.loop.run_until_complete(
+            asyncio.create_subprocess_shell(blocking_shell_command, stdout=asyncio.subprocess.PIPE,
+            creationflags=creationflags)
+        )
+        self.loop.run_until_complete(asyncio.sleep(1))
+        if sys.platform == 'win32':
+            proc.send_signal(signal.CTRL_BREAK_EVENT)
+        proc.kill()
```

Comment:
Do we _also_ need this line on Windows? It seems to eventually boil down to `self.send_signal(signal.SIGKILL)` (the very last line of the toplevel `subprocess.py` :-).